systemlog/systemlog.go

```diff
@@ -121,7 +121,7 @@ func (x *Indexer) PutBlock(_ context.Context, blk *block.Block) error {
 
 		evmTransferList := &iotextypes.EvmTransferList{}
 		for _, l := range receipt.Logs {
-			if action.IsSystemLog(l) {
+			if action.IsEvmTransfer(l) {
```

Comment:
do i need to write the withdraw bucket systemlog, or rosetta client will switch to directly read the new bucket in chain.db?

Reply:
not sure what you mean. rosetta client will only read through the new API you expose. So it can only read whatever  and however you expose through those APIs